types/opossum/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for opossum 6.2
+// Type definitions for opossum 6.3
```

Comment:
This new version doesn't seem right; the version that was just released was 8.1, no?

Reply:
Sorry, that's my bad. Just made the change!